packages/core-commands/src/site-editor-navigation-commands.js

```diff
@@ -258,83 +263,105 @@ function useSiteEditorBasicNavigationCommands() {
 		'site-editor.php'
 	);
 	const isTemplatesAccessible = useIsTemplatesAccessible();
+	const isTemplatesEditable = useIsTemplatesEditable();
```

Comment:
Based on [this discussion](https://github.com/WordPress/gutenberg/pull/61444#discussion_r1592841651), it seems better not to create custom hooks. You will only need to check whether it is possible to create a template, as shown below.

```javascript
const canCreateTemplate = useSelect( ( select ) => {
	return select( coreStore ).canUser( 'create', 'templates' );
}, [] );
```

Reply:
Thanks - this makes sense and uses the same `canCreateTemplate` value to determine whether the user should be sent to the classic admin screen for patterns or to the site editor's Pattern screen.